src/clusterfuzz/_internal/issue_management/google_issue_tracker/issue_tracker.py

```diff
@@ -799,6 +817,37 @@ def new_issue(self):
     }
     return Issue(data, True, self)
 
+  def get_relative_component_path(self, component_id):
```

Comment:
Can we make this private? i.e. name this `_get_relative_component_path`.

Reply:
Done.